source/extensions/filters/network/mysql_proxy/mysql_session.h

```diff
@@ -11,20 +11,20 @@ namespace MySQLProxy {
 class MySQLSession : Logger::Loggable<Logger::Id::filter> {
 public:
   enum class State {
-    MYSQL_INIT = 0,
-    MYSQL_CHALLENGE_REQ = 1,
-    MYSQL_CHALLENGE_RESP_41 = 2,
-    MYSQL_CHALLENGE_RESP_320 = 3,
-    MYSQL_SSL_PT = 4,
-    MYSQL_AUTH_SWITCH_REQ = 5,
-    MYSQL_AUTH_SWITCH_REQ_OLD = 6,
-    MYSQL_AUTH_SWITCH_RESP = 7,
-    MYSQL_AUTH_SWITCH_MORE = 8,
-    MYSQL_REQ_RESP = 9,
-    MYSQL_REQ = 10,
-    MYSQL_RESYNC = 11,
-    MYSQL_NOT_HANDLED = 12,
-    MYSQL_ERROR = 13,
+    Init = 0,
+    ChallengeReq = 1,
+    ChallengeResp41 = 2,
+    ChallengeResp320 = 3,
+    SslPt = 4,
+    AuthSwitchReq = 5,
+    AuthSwitchReqOld = 6,
+    AuthSwitchResp = 7,
+    AuthSwitchMore = 8,
+    ReqResp = 9,
+    Req = 10,
+    Resync = 11,
+    NotHandled = 12,
+    Error = 13,
```

Comment:
FWIW, here and a few other places I took the liberty of removing what appeared to be redundant prefixing to me. For example, in this case since we're already in the `MySQLProxy` namespace and in the `MySQLSession` class, I figured the `MYSQL_` prefix here is unnecessary.

Reply:
Yeah, I noticed those, and I personally agree with your decisions.